metadata-ingestion/setup.py

```diff
@@ -321,7 +321,6 @@ def get_long_description():
     "pytest-asyncio>=0.16.0",
     "pytest-cov>=2.8.1",
     "pytest-docker>=0.10.3,<0.12",
-    "tox",
```

Comment:
tried parallelising via tox but there was some problem during builds. the build wheel being generated by the 3 test suites was stepping on each other's toes and causing at least one of them to fail. Also, thought it might be better to do it via github actions so we can easily parallelise and isolate. Now if that SAP HANA test flakes we can simply re-run that (that and nifi one under slow integration) one instead of all integration tests.